src/lib/command.c

```diff
@@ -454,9 +457,9 @@ static SrnRet srn_command_parse(SrnCommand *cmd, void *user_data){
             }
         }
         narg++;
-        if (!ptr){
-            goto missing_arg;
-        }
+    }
+    if (!ptr){
+        goto missing_arg;
```

Comment:
I think this check can be omitted.

Reply:
That was also my feeling, but I wasn't sure. Done